src/refract/RenderJSONVisitor.cc

```diff
@@ -61,8 +61,8 @@ namespace refract
                     continue;
                 }
 
-                if ((*it)->element() == "ref") {
-                    HandleRefWhenFetchingMembers<T>(*it, members, FetchMembers<T>);
+                if (RefElement* ref = TypeQueryVisitor::as<RefElement>(*it)) {
```

Comment:
Probably because 

https://github.com/apiaryio/drafter/pull/476/files#diff-b5d89872d6f93c63ff1667145fb3d8a9L240

You can not remove Expanding Reference